pkg/volume/csi/csi_block.go

```diff
@@ -129,7 +129,7 @@ func (m *csiBlockMapper) getPublishPath() string {
 // returns: pods/{podUID}/volumeDevices/kubernetes.io~csi, {specName}
 func (m *csiBlockMapper) GetPodDeviceMapPath() (string, string) {
 	path := m.plugin.host.GetPodVolumeDeviceDir(m.podUID, utilstrings.EscapeQualifiedName(CSIPluginName))
-	klog.V(4).Infof(log("blockMapper.GetPodDeviceMapPath [path=%s; name=%s]", path, m.specName))
+	klog.V(4).InfoS(log("BlockMapper.GetPodDeviceMapPath"), "path", path, "name", m.specName)
```

Comment:
Can you change the key `name` to show what object this name belongs to? Is this a `blockMapperName` a `volumeName` etc